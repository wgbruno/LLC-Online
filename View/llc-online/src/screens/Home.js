import React from "react";
import Footer from "../components/Footer";
import Gallery from "../components/Gallery";
import Hero from "../components/Hero";
import Services from "../components/Services"
import About from "../components/About"
import InventoryRecord from "../components/InventoryRecord"

export default function Home(){
    return (
      <div>    
          <Hero />
          <About />
          <Services />
          <Gallery />
          <Footer />
      </div>
    );
}