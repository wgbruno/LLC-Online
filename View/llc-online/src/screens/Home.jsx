import React from "react";

export default function Home(){
    return (
        <>
            <div>
          <meta charSet="utf-8" />
          <meta content="width=device-width, initial-scale=1.0" name="viewport" />
          <title>Butterfly Bootstrap Template - Index</title>
          <meta content name="description" />
          <meta content name="keywords" />
          {/* Favicons */}
          <link href="assets/img/favicon.png" rel="icon" />
          <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon" />
          {/* Google Fonts */}
          <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Dosis:300,400,500,,600,700,700i|Lato:300,300i,400,400i,700,700i" rel="stylesheet" />
          {/* Vendor CSS Files */}
          <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet" />
          <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet" />
          <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet" />
          <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet" />
          <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet" />
          {/* Template Main CSS File */}
          <link href="assets/css/style.css" rel="stylesheet" />
          {/* =======================================================
    * Template Name: Butterfly - v4.10.0
    * Template URL: https://bootstrapmade.com/butterfly-free-bootstrap-theme/
    * Author: BootstrapMade.com
    * License: https://bootstrapmade.com/license/
    ======================================================== */}
          
          <main id="main">
            {/* ======= About Section ======= */}
            <section id="about" className="about">
              <div className="container">
                <div className="row">
                  <div className="col-xl-5 col-lg-6 d-flex justify-content-center video-box align-items-stretch position-relative">
                    <a href="https://www.youtube.com/watch?v=jDDaplaOz7Q" className="glightbox play-btn mb-4" />
                  </div>
                  <div className="col-xl-7 col-lg-6 icon-boxes d-flex flex-column align-items-stretch justify-content-center py-5 px-lg-5">
                    <h3>Enim quis est voluptatibus aliquid consequatur fugiat</h3>
                    <p>Esse voluptas cumque vel exercitationem. Reiciendis est hic accusamus. Non ipsam et sed minima temporibus laudantium. Soluta voluptate sed facere corporis dolores excepturi. Libero laboriosam sint et id nulla tenetur. Suscipit aut voluptate.</p>
                    <div className="icon-box">
                      <div className="icon"><i className="bx bx-fingerprint" /></div>
                      <h4 className="title"><a href>Lorem Ipsum</a></h4>
                      <p className="description">Voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident</p>
                    </div>
                    <div className="icon-box">
                      <div className="icon"><i className="bx bx-gift" /></div>
                      <h4 className="title"><a href>Nemo Enim</a></h4>
                      <p className="description">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque</p>
                    </div>
                    <div className="icon-box">
                      <div className="icon"><i className="bx bx-atom" /></div>
                      <h4 className="title"><a href>Dine Pad</a></h4>
                      <p className="description">Explicabo est voluptatum asperiores consequatur magnam. Et veritatis odit. Sunt aut deserunt minus aut eligendi omnis</p>
                    </div>
                  </div>
                </div>
              </div>
            </section>{/* End About Section */}
           
            
            {/* ======= Gallery Section ======= */}
            <section id="gallery" className="gallery">
              <div className="container">
                <div className="section-title">
                  <h2>Gallery</h2>
                  <p>Magnam dolores commodi suscipit. Necessitatibus eius consequatur ex aliquid fuga eum quidem. Sit sint consectetur velit. Quisquam quos quisquam cupiditate. Et nemo qui impedit suscipit alias ea. Quia fugiat sit in iste officiis commodi quidem hic quas.</p>
                </div>
                <div className="row no-gutters">
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-1.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-1.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-2.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-2.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-3.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-3.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-4.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-4.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-5.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-5.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-6.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-6.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-7.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-7.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                  <div className="col-lg-3 col-md-4">
                    <div className="gallery-item">
                      <a href="assets/img/gallery/gallery-8.jpg" className="galleery-lightbox" data-gallery="gallery-item">
                        <img src="assets/img/gallery/gallery-8.jpg" alt="" className="img-fluid" />
                      </a>
                    </div>
                  </div>
                </div>
              </div>
            </section>{/* End Gallery Section */}
        
          </main>{/* End #main */}
          <a href="#" className="back-to-top d-flex align-items-center justify-content-center"><i className="bi bi-arrow-up-short" /></a>
          {/* Vendor JS Files */}
          {/* Template Main JS File */}
        </div>
        </>
    );
}