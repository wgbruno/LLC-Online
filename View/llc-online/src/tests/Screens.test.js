import {expect, test} from '@jest/globals';
import renderer from 'react-test-renderer';
import Home from "../screens/Home"
import LoginPage from '../screens/loginpage';

test("HomePage", function (){
    const homepage = renderer.create(<Home />).toJSON();
    expect(homepage).toMatchSnapshot();
});

test("Login", function (){
    const login = renderer.create(<LoginPage />).toJSON();
    expect(login).toMatchSnapshot();
});

