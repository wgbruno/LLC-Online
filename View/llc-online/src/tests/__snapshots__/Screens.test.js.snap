// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login 1`] = `
<div
  className="LoginPage"
>
  <div
    className="auth-form-container"
  >
    <h2>
      Login
    </h2>
    <form
      className="login-form"
      onSubmit={[Function]}
    >
      <label
        htmlFor="email"
      >
        email
      </label>
      <input
        id="email"
        name="email"
        onChange={[Function]}
        placeholder="youremail@gmail.com"
        type="email"
        value=""
      />
      <label
        htmlFor="password"
      >
        password
      </label>
      <input
        id="password"
        name="password"
        onChange={[Function]}
        placeholder="********"
        type="password"
        value=""
      />
      <button
        type="submit"
      >
        Log In
      </button>
    </form>
    <button
      className="link-btn"
      onClick={[Function]}
    >
      Don't have an account with LLC? Register here.
    </button>
  </div>
</div>
`;
